CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG
 
+## v1.0.0 2020-10-24
+- bed file as String, minor changes
+
+## Unreleased
```

Comment:
Could remove this line, as "Upgrade for Vidarr" will be included in v1.0.0

Reply:
Ok, I can add hg38 test np